src/inference/src/dev/core_impl.cpp

```diff
@@ -580,6 +580,8 @@ ov::Plugin ov::CoreImpl::get_plugin(const std::string& pluginName) const {
     auto deviceName = pluginName;
     if (deviceName == ov::DEFAULT_DEVICE_NAME)
         deviceName = "AUTO";
+    if (deviceName == "(CPU)")
+        deviceName = "CPU";
```

Comment:
do we need the same for other devices? Like `(GPU)` etc

Reply:
Currently only the CPU has this alias(CPU). When the CPU is used as an acceleration.